docs/user/OAuth2OIDC-VMware-cloud-services.md

```diff
@@ -0,0 +1,55 @@
+# VMware Cloud Services
```

Comment:
I'd change the title to "Configuring XXX as an OIDC provider" (using just the provider name as the title could be misleading, no?